lib/rubocop/cop/gemspec/ruby_version_globals_usage.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Gemspec
+      # Checks that `RUBY_VERSION` constant is not used in gemspec
```

Comment:
I think we need to explain why we cannot use RUBY_VERSION in gemspec.
Can you write the explanation?